pkg/cryptoauthlib/include/atca_params.h

```diff
@@ -44,9 +51,19 @@ extern "C" {
  * @{
  */
 
-#ifndef ATCA_PARAM_I2C
-#define ATCA_PARAM_I2C           I2C_DEV(0)
+#ifndef ATCA_PARAM_I2C_DEV0
+/**
+ * @brief   ATCA device connected to I2C bus 0
+ */
+#define ATCA_PARAM_I2C_DEV0      I2C_DEV(0)
 #endif
+
+#if IS_USED(MODULE_PSA_SECURE_ELEMENT_MULTIPLE)
```

Comment:
Looks odd seeing this. Why would a file called `acta_params.h` care about the amount of secure elements? Could you elaborate?

Reply:
I know what you mean. 
I added this to have the possibility to use multiple SEs in `examples/psa_crypto` and to kind of show how this could be done. 
I don't really have an idea, yet, how to do this differently, I'll have to think about it.

Reply:
How much of a use-case are multiple secure elements to begin with?
*[YAGNI](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it)*

Reply:
Hmmm it may not be important to people who use SEs for fun at home or for research, but I think it is a valid use case. At least the PSA specification includes it and it's an issue for the MbedTLS people.
But maybe it'll suffice to describe it in detail in the documentation.

Reply:
I removed all traces of the second secure element and will update the documentation